### PR TITLE
Miscellaneous improvements of german translation

### PR DESCRIPTION
**Pull request by Eskaan**

### CODE REVIEW

**Comment by Eskaan** [comment]
Yeah just seeing you are working on it already

File: de.json [hunk 1]
```diff
@@ -14,31 +14,38 @@
...
+				"ru": "Russisch",
+				"zh": "Chinesisch",
+				"zh-CN": "Vereinfachtes Chinesisch"
 			}
 		}
 	},
 	"error": {
 		"status": "{status} Fehler",
 		"message": {
-			"404": "Oh oh, sieht aus als hättest du dich im [Labyrinth](https://github.com/modrinth/labrinth) verlaufen.<br />Die Seite nach der du gesucht hast, wurde nicht gefunden.",
```
Comment (mooziii) [inline de.json:36]: Why did you remove that?

File: de.json [hunk 2]
```diff
@@ -14,31 +14,38 @@
...
+			"404": "Oh oh, sieht aus als hättest du dich im [Labyrinth](https://github.com/modrinth/labrinth) verlaufen. Die Seite nach der du gesucht hast, wurde nicht gefunden.",
 			"project_missing": "Das {type} nach dem du gesucht hast, wurde nicht gefunden.",
 			"default": "Ein unerwarteter Fehler ist aufgetreten."
 		},
 		"actions": {
-			"home": "Gehe zur Startseite",
+			"home": "Zur Startseite",
```
Comment (mooziii) [inline de.json:48]: I think "Zurück zur Startseite" would be better because this is displayed on the 404 page

File: de.json [hunk 3]
```diff
@@ -92,7 +99,7 @@
 			"client_side": "Client-Seite",
 			"values": {
 				"required": "Benötigt",
-				"unsupported": "Inkompatibel",
```
Comment (mooziii) [inline de.json:95]: Isn't that basically the same?

File: de.json (hunk 2 above)
Comment (Eskaan) [reply de.json:48]: The english translation is Go home so I translated that. Go back home would be Zurück zur Startseite. Just wanting to keep future compatability

File: de.json (hunk 1 above)
Comment (Eskaan) [reply de.json:36]: Sry one sec

**Comment by mooziii** [comment]
Will do more stuff later, im tired and gonna go sleep

File: de.json (hunk 3 above)
Comment (Eskaan) [reply de.json:95]: Not really, its the differnence betwenn won't work and we don't know

**Comment by Eskaan** [comment]
Understandable at half past 12
